core/vm/evm.go

```diff
@@ -492,9 +495,8 @@ func (evm *EVM) create(caller ContractRef, codeAndHash *codeAndHash, gas uint64,
 	}
 
 	var ret []byte
-	if err == nil {
-		ret, err = evm.interpreter.Run(contract, nil, false)
-	}
+	var err error
+	ret, err = evm.interpreter.Run(contract, nil, false)
```

Comment:
```suggestion
	ret, err := evm.interpreter.Run(contract, nil, false)
```

(and also remove the `var ret []byte` above